components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -278,6 +276,16 @@ - (NSString *)accessibilityLabelWithTitle:(NSString *)title {
   return [labelComponents componentsJoinedByString:@", "];
 }
 
+- (CGPoint)badgeCenterFrom:(CGRect)iconFrame isRTL:(BOOL)isRTL {
+  if (isRTL) {
+    return CGPointMake(CGRectGetMinX(iconFrame),
+                       CGRectGetMinY(iconFrame) + kMDCBottomNavigationItemViewBadgeYOffset);
+  } else {
```

Comment:
Nit: Please remove the `else` clause and just let the second `return` statement be on its own after the `if` block.